pkg/notes/notes.go

```diff
@@ -484,7 +485,9 @@ func (g *Gatherer) ReleaseNoteFromCommit(result *Result) (*ReleaseNote, error) {
 	}
 
 	// Uppercase the first character of the markdown to make it look uniform
-	markdown = strings.ToUpper(string(markdown[0])) + markdown[1:]
+	r := []rune(markdown)
+	r[0] = unicode.ToUpper(r[0])
+	markdown = string(r)
```

Comment:
Let's make this a dedicated function with tests and re-use it in the other places.

Reply:
Thank you for your kind feedback @saschagrunert!
I just pushed some changes, please take a look.